File: fixtures/projects/clang-native/src/Sample.m
@interface Sample
- (int)classify:(int)left other:(int)right;
@end

@implementation Sample
- (int)classify:(int)left other:(int)right {
  if (left > 0 && right > 0) {
    return left + right;
  }
  return 0;
}
@end

File: fixtures/minimal/rust/sample.rs
pub fn classify(value: i32) -> &'static str {
    if value > 10 && value < 100 {
        "medium"
    } else if value >= 100 {
        "large"
    } else {
        "small"
    }
}

pub fn enabled(left: bool, right: bool) -> bool {
    left && right
}
